app/_locales/zh_CN/messages.json

```diff
@@ -1128,7 +1128,7 @@
     "description": "Return the user to the site that initiated onboarding"
   },
   "onlyAddTrustedNetworks": {
-    "message": "恶意的 Ethereum 以太坊网络提供商可以伪造区块链状态，并记录您的网络活动。只添加您信任的自定义网络。"
+    "message": "恶意的 以太坊网络提供商可以伪造区块链状态，并记录您的网络活动。只添加您信任的自定义网络。"
```

Comment:
When testing this in Google Translate, "Ethereum" still appears in the output :thinking:

Reply:
I'm not getting the same results, 
![image](https://user-images.githubusercontent.com/4448075/110352940-1e31d300-7ffc-11eb-8b74-d37733a5f82d.png)

Reply:
malicious blockchains, tho.

Reply:
I think you're translating a different locale here (this is simplified Chinese, not Japanese)

Reply:
Oops, i was on the wrong locale.